codegens/curl/lib/index.js

```diff
@@ -186,6 +187,14 @@ self = module.exports = {
         description: 'Set a character used to mark the continuation of a statement on the next line ' +
           '(generally, \\ for OSX/Linux, ^ for Windows)'
       },
+      {
+        name: 'Quote Type',
+        id: 'quoteType',
+        availableOptions: ['single', 'double'],
+        type: 'string',
+        default: 'single',
+        description: 'String denoting the quote type to use (single or double) for URL'
```

Comment:
Do you think we should also specify when single and double quote are preferable like for line continuation `(generally, ...)` ? @webholik